Dockerfile

```diff
@@ -9,15 +9,17 @@
 #
 ###
 
-FROM alpine:3.2
+FROM alpine:latest
 MAINTAINER Kingsquare <docker@kingsquare.nl>
+MAINTAINER bubak4 <mar@centrum.cz>
```

Comment:
We might want to remove the maintainers as a whole .. see docker/docker#25466
Possibly include a label referring to this repo ... that has more info anyway on the subject (code authors etc)